# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/ipc/APZChild.cpp
# Commit: 76279f0a6d33
# Full Hash: 76279f0a6d33d6fe128a9f3aabe5393e52181818
# Author: James Willcox <snorp@snorp.net>
# Date: 2020-06-08 21:38:11
# Regressor Bug: 1536833
# File Overlap Count: 1
# Description:
#   Bug 1536833 - Fix overscroll effect on Android r=botond,geckoview-reviewers,agi
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D78130
# ==============================================================================

diff -r ad757ab87752 -r 76279f0a6d33 gfx/layers/ipc/APZChild.cpp
--- a/gfx/layers/ipc/APZChild.cpp	Sun Jun 07 23:15:45 2020 +0000
+++ b/gfx/layers/ipc/APZChild.cpp	Mon Jun 08 17:13:28 2020 +0000
@@ -42,14 +42,16 @@
 }
 
 mozilla::ipc::IPCResult APZChild::RecvUpdateOverscrollVelocity(
-    const float& aX, const float& aY, const bool& aIsRootContent) {
-  mController->UpdateOverscrollVelocity(aX, aY, aIsRootContent);
+    const ScrollableLayerGuid& aGuid, const float& aX, const float& aY,
+    const bool& aIsRootContent) {
+  mController->UpdateOverscrollVelocity(aGuid, aX, aY, aIsRootContent);
   return IPC_OK();
 }
 
 mozilla::ipc::IPCResult APZChild::RecvUpdateOverscrollOffset(
-    const float& aX, const float& aY, const bool& aIsRootContent) {
-  mController->UpdateOverscrollOffset(aX, aY, aIsRootContent);
+    const ScrollableLayerGuid& aGuid, const float& aX, const float& aY,
+    const bool& aIsRootContent) {
+  mController->UpdateOverscrollOffset(aGuid, aX, aY, aIsRootContent);
   return IPC_OK();
 }
 